Makefile

```diff
@@ -51,15 +51,15 @@ libfuzzer honggfuzz:
 	    --manifest-path examples/basic/Cargo.toml \
 	    --runs 100000 \
 	    --engine $@ \
-	    --release \
+	    --profile release \
```

Comment:
Can you update the examples to include a `fuzz` profile, instead? The bolero workspace should probably also include one as well.

Reply:
Done, thank you for the idea :)

Reply:
All of the tests should be updated to use the `fuzz` profile. And then we're good to merge :)

Reply:
Oh right, thank you! I just pushed a commit removing the `--profile` argument altogether, as bolero defaults to the `fuzz` profile :)

Also just in case, for #161, I’m waiting for your feedback on whether the BOLERO_LIBTEST_HARNESS environment variable is a red herring or not, and on whether sometimes libfuzzer should be able to run the binary without going through the libtest harness :)